cmd/build/helmify/static/README.md

```diff
@@ -114,6 +114,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | enableDeleteOperations                        | Enable validating webhook for delete operations. Does not work with `validatingWebhookCustomRules`                                                                                                                                                  | `false`                                                                   |
 | enableExternalData                            | Enable external data (alpha feature)                                                                                                                                                                                                                | `false`                                                                   |
 | enableTLSHealthcheck                          | Enable probing webhook API with certificate stored in certDir                                                                                                                                                                                       | `false`                                                                   |
+| maxServingThreads                             | Limit the number of concurrent calls the the validation backend made by the validation webhook. -1 limits this value to GOMAXPROCS. Lowering this value may lower max RAM usage and limit CPU throttling, Tuning it can optimize serving capacity.  | `-1`                                                                      | 
```

Comment:
```suggestion
| maxServingThreads                             | Limit the number of concurrent calls the validation backend made by the validation webhook. -1 limits this value to GOMAXPROCS. Configuring this value may lower max RAM usage and limit CPU throttling, Tuning it can optimize serving capacity.  | `-1`                                                                      | 
```
I'm not sure how `Lowering` can be interpreted here since the default is -1. when we say `lower` what is that compared to?